packages/ecmascript/src/Module.ts

```diff
@@ -94,6 +205,7 @@ export interface ModuleElementList {
 		return this;
 	};
 
+	/** @deprecated - unused and not part of {@link ModuleExport} */
```

Comment:
Deprecations should happen after an internal triage, lets add this to the roadmap to discuss?